module/os/linux/zfs/vdev_disk.c

```diff
@@ -777,9 +775,6 @@ vbio_submit(vbio_t *vbio, abd_t *abd, uint64_t size)
 	vdev_submit_bio(vbio->vbio_bio);
```

Comment:
Can you add a comment indicating that `vbio` can't be touched after this point or something to that affect?

Reply:
Good call, done!